.mailmap

```diff
@@ -1464,6 +1464,7 @@ klaasvanaarsen <44929042+klaasvanaarsen@users.noreply.github.com>
 lastcodestanding <rohang71604@gmail.com>
 latot <felipematas@yahoo.com>
 luzpaz <luzpaz@users.noreply.github.com> luz.paz <luzpaz@users.noreply.github.com>
+maciejskorski <maciej.skorski@gmail.com>
```

Comment:
Is this how you would like your name to be recorded in the AUTHORS file?

If you were to add an entry like
```
Maciej Skorski <maciej.skorski@gmail.com> maciejskorski <maciej.skorski@gmail.com>
```
then your name in the AUTHORS file would be recorded as
```
Maciej Skorski <maciej.skorski@gmail.com>
```

Reply:
thanks a lot, yes I was wondering how to put it that way!